craco.config.cjs

```diff
@@ -69,7 +79,13 @@ module.exports = {
     },
   },
   webpack: {
-    plugins: [new VanillaExtractPlugin({ identifiers: 'short' })],
+    plugins: [
+      // react-markdown requires process, and Webpack 5 does not polyfill, so we provide it explicitly.
+      new ProvidePlugin({ process: 'process/browser' }),
+      // vanilla-extract has poor performance on M1 machines with 'debug' identifiers.
```

Comment:
nit: It would be great if the comment explained the difference between `debug` and `short`.